apps/worker/src/.env.test

```
@@ -97,3 +97,4 @@ IS_STEP_RUN_LOGS_WRITE_ENABLED=true
 IS_WORKFLOW_RUN_LOGS_WRITE_ENABLED=true
 IS_SUBSCRIBERS_SCHEDULE_ENABLED=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IS_SUBSCRIBERS_SCHEDULE_ENABLED key should go before the IS_TRACE_LOGS_ENABLED key

(UnorderedKey)